README.md

```diff
@@ -15,7 +15,8 @@
 English |
 [Русский](README_ru-RU.md) |
 [简体中文](README_zh-CN.md) |
-[正體中文](README_zh-TW.md)
+[正體中文](README_zh-TW.md) |
+[日本語](README_ja-JP.md)
```

Comment:
The languages are sorted by the language code, so put it between English and Russian.

Reply:
@ouuan Thanks! Fixed it.